packages/jupyter-chat/src/model.ts

```diff
@@ -328,6 +337,18 @@ export abstract class AbstractChatModel implements IChatModel {
     localStorage.setItem(`@jupyter/chat:${this._id}`, JSON.stringify(storage));
   }
 
+  /**
+   * Promise that resolves when the model is ready.
+   */
+  readonly ready: Promise<void>;
```

Comment:
```suggestion
  get ready(): Promise<void> {
    return this._readyDelegate.promise;
  }
```
We could use a getter function.